Dockerfile

```diff
@@ -18,7 +18,7 @@
 ARG ARCH
 
 # Build the manager binary
-FROM golang:1.22 as builder
+FROM golang:1.22 AS builder
```

Comment:
I keep thinking I should do this whenever I see that warning in the logs. Thanks!